docs/architecture/adrs.rst

```diff
@@ -0,0 +1,16 @@
+Architecture Decision Records
+*****************************
+
+These are detailed records of technical decisions made in the past
+regarding GeoTrellis.
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Index
+   :glob:
```

Comment:
Notice this `toctree` isn't marked as `hidden`, so it will also render in this "landing page".